### PR TITLE
[WIP] Add some text to the README

### PR DESCRIPTION
**Pull request by skade**
Here's some updates to the README.

I would like to fill the resource section a little, can you share your favourite ones? /cc @rust-community/events-team

### CODE REVIEW

**Review by mattgathu — APPROVED** [review]
🍭

File: README.md [hunk 1]
```diff
@@ -1,15 +1,27 @@
...
 - **Meetup Checklist:** [Here](https://github.com/rust-community/events-team/blob/master/standards/MEETUP_TALKS.md)
 
+## 📚 Resources
+
+Our favourite resources around events.
+
+- [**OpenTechSchool Meetup Guide**](http://www.opentechschool.org/handbooks/workshops/organisers-schedule.html): A great guide on how to set up "Hack & Learn" style meetups
+- [**OpenTechSchool Coaching Guide**](http://opentechschool.github.io/slides/presentations/coaching/): A great resource on what to do when teaching people.
+- [**rampup.me**](https://rampup.me): Information site on how to run accessible events.
+- [**Conference Katamari**](http://conferencekatamari.info/): The rule of six: how to go out after an event.
```
Comment (JohnTitor) [inline README.md]:
```suggestion
- [**Conference Katamari**](https://conferencekatamari.info/): The rule of six: how to go out after an event.
```

File: README.md [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
 
 ## 💡 Guides
 
 - **Meetup Checklist:** [Here](https://github.com/rust-community/events-team/blob/master/standards/MEETUP_TALKS.md)
 
+## 📚 Resources
+
+Our favourite resources around events.
+
+- [**OpenTechSchool Meetup Guide**](http://www.opentechschool.org/handbooks/workshops/organisers-schedule.html): A great guide on how to set up "Hack & Learn" style meetups
```
Comment (JohnTitor) [inline README.md]:
```suggestion
- [**OpenTechSchool Meetup Guide**](https://www.opentechschool.org/handbooks/workshops/organisers-schedule.html): A great guide on how to set up "Hack & Learn" style meetups
```

File: README.md [hunk 3]
```diff
@@ -1,15 +1,27 @@
...
+- [**OpenTechSchool Coaching Guide**](http://opentechschool.github.io/slides/presentations/coaching/): A great resource on what to do when teaching people.
```
Comment (JohnTitor) [inline README.md]:
```suggestion
- [**OpenTechSchool Coaching Guide**](https://opentechschool.github.io/slides/presentations/coaching/): A great resource on what to do when teaching people.
```

File: README.md [hunk 4]
```diff
@@ -1,15 +1,27 @@
...
+- [**rampup.me**](https://rampup.me): Information site on how to run accessible events.
```
Comment (JohnTitor) [inline README.md]:
```suggestion
- [**rampup.me**](https://ramp-up.me): Information site on how to run accessible events.
```

**Review by JohnTitor — COMMENTED** [review]
Add minor suggestions (I hope this pull request is still valid...)

**Review by JohnTitor — APPROVED** [review]
Rebased to resolve conflicts. @skade Now, we can merge this?

**Comment by skade** [comment]
I'll merge it, thanks!

**Pull request merged**